src/pages/UserDashboard/ItemDetail/ItemDetail.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import Bike from '../../../components/asset/bike.jpg';
+import DisplayItemCard from '../../../components/Card/DisplayItemCard';
+
+const ItemDetail = () => {
+  const { id } = useParams();
+  const cars = useSelector((state) => state.cars.cars);
+  const numericId = parseInt(id, 10);
+  const car = cars.find((car) => car.id === numericId);
+  // const {
+  //   name, description, pricePerHr, seatingCapacity,
+  // } = car;
```

Comment:
-[ ] Kindly, delete the old commented code present in this file in order to preserve the repo's cleanliness. 👍